scripts/validate_docstrings.py

```diff
@@ -471,6 +471,36 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        i = list(self.doc_parameters).index(param)
+        desc_list = [line
+                     for line in self.doc._parsed_data['Parameters'][i][-1]
+                     if line]
+        # Find and strip out any sphinx directives
+        for directive in DIRECTIVES:
+            full_directive = '.. {}'.format(directive)
+            for desc_item in desc_list:
+                if full_directive in desc_item:
+                    # Only retain any description before the directive
+                    desc_list = desc_list[:desc_list.index(desc_item)]
+                    break
+        return desc_list
+
+    def parameter_correct_end(self, param):
+        if self.parameter_desc(param)[-1] == '.':
+            return True
+        else:
+            # Return True if a parameter description ends
+            # with a bullet point
+            for desc_line in self.parameter_desc_list(param)[::-1]:
```

Comment:
When would this ever make more than one iteration?

Reply:
It will make more iterations when a parameter description ends with a long sentence in a bullet point that require more lines.

E.g.
If a parameter description is
```
Parameters
----------
method : {'min', 'max'}, default 'min
    
    ...

    * 'max': the description for max that can also be very long and
      requires extra rows because PEP8 does not allow a line that
      is longer than 79 chars
```

The list returned by `self.parameter_desc_list(param)` is:
```
[
...
"* 'max': the description for max that can also be very long and",
"  requires extra rows because PEP8 does not allow a line that",
"  is longer than 79 chars"
]
```